fix ftpwho install dir

Upstream-Status: Pending

ftpwho is installed into /usr/bin/, not /usr/sbin, so fix it

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 policy/modules/contrib/ftp.fc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/contrib/ftp.fc b/policy/modules/contrib/ftp.fc
index ddb75c1..26fec47 100644
--- a/policy/modules/contrib/ftp.fc
+++ b/policy/modules/contrib/ftp.fc
@@ -9,7 +9,7 @@
 
 /usr/kerberos/sbin/ftpd	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
 
-/usr/sbin/ftpwho	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
+/usr/bin/ftpwho	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
 /usr/sbin/in\.ftpd	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
 /usr/sbin/muddleftpd	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
 /usr/sbin/proftpd	--	gen_context(system_u:object_r:ftpd_exec_t,s0)
-- 
1.7.10.4

